pkg/commands/start_test.go

```diff
@@ -125,6 +129,7 @@ func dnsLookup(t *testing.T, resolverPort, resolverProtocol, domain string) ([]n
 	return resolver.LookupIP(ctx, "ip4", domain)
 }
 
+//nolint:gosec
```

Comment:
Do you recall _why_ `gosec` thought this was an issue?

Reply:
This is removed in the bigger PR and should no longer be the case. Basically, linter doesn't like weak random number generators which doesn't make sense in this case.

Reply:
Sounds good.